tests/integration-tests/tests/spot/test_spot/test_spot_default/pcluster.config.ini

```diff
@@ -0,0 +1,19 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
```

Comment:
you forgot compute and master instance type

Reply:
Yep sorry